web_confirm_window_close/static/src/js/web_confirm_window_close.js

```diff
@@ -10,6 +10,7 @@ odoo.define('web_confirm_window_close', function (require) {
             if ($('html').find('.oe_form_dirty').length) {
                 e.preventDefault();
             }
+            return 'Your have unsaved data. Are you sure you want to close window?';
```

Comment:
Please use translations

Reply:
@Tardo  I came to know about Chrome & Firefox has removed the custom message on window close event...
Chrome
https://www.chromestatus.com/feature/5349061406228480

Firefox
https://developer.mozilla.org/en-US/docs/Web/Events/beforeunload

So shall I add a custom message in return statement? Or returning blank string will be fine...

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/WindowEventHandlers/onbeforeunload#Notes
>Starting with Firefox 44, Chrome 51, Opera 38 and Safari 9.1, a generic string not under the control of the webpage will be shown instead of the returned string.

I don't know what is the criteria to keep maintaining features that works on outdated browsers.

Any suggestion @OCA/web-maintainers ?

Reply:
Well, this is a very specific case that can be determined by the maintainers of the module.

Reply:
@pedrobaeza  @Tardo  @OCA/web-maintainers I checked this PR with @devang-dreambits  yesterday and what we found was that returning any string is enough to invoke the Confirm Dialog in chrome. 
In outdated browsers, both chrome and firefox did show the returned string in the dialog but that it not an requirement.
So wouldn't it be better to return an empty string as it would work well with both new and old browsers and also not get into extra effort of making it translatable as it would be ignored by browsers anyways

Reply:
OK for me.